### PR TITLE
Add docs for sending metrics to SigNoz Cloud with OTEL Collector

### PR DESCRIPTION
**Pull request by srikanthccv**

### CODE REVIEW

File: docs/userguide/send-metrics-cloud.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    scrapers:
+      cpu: {}
+      disk: {}
+      load: {}
+      filesystem: {}
+      memory: {}
+      network: {}
+      paging: {}
+      process:
+       mute_process_name_error: true
```
Comment (prashant-shahi) [inline docs/userguide/send-metrics-cloud.md:47]:
```suggestion
       mute_process_name_error: true
       mute_process_exe_error: true
       mute_process_io_error: true
```

File: docs/userguide/send-metrics-cloud.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
```
Comment (prashant-shahi) [inline docs/userguide/send-metrics-cloud.md:109]:
```suggestion
       mute_process_name_error: true
       mute_process_exe_error: true
       mute_process_io_error: true
```

same as above

**Review by ankit01-oss — APPROVED** [review]

File: docs/userguide/send-metrics-cloud.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+      processes: {}
+  rabbitmq:
+    endpoint: http://localhost:15672
+    username: <RABBITMQ_USERNAME>
+    password: <RABBITMQ_PASSWORD>
+    collection_interval: 10s
```
Comment (prashant-shahi) [inline docs/userguide/send-metrics-cloud.md:115]: isn't `10s` too frequent for collection interval?

File: docs/userguide/send-metrics-cloud.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+processors:
+  batch:
+    send_batch_size: 1000
+    timeout: 10s
+  # Ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md
+  resourcedetection:
+    detectors: [env, system, ec2] # include ec2 for AWS, gce for GCP and azure for Azure.
```
Comment (prashant-shahi) [inline docs/userguide/send-metrics-cloud.md:122]:
```suggestion
    detectors: [env, system, ec2] # include ec2 for AWS, gcp for GCP and azure for Azure.
```

File: docs/userguide/send-metrics-cloud.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+      hostname_sources: [os] # alternatively, use [dns,os] for setting FQDN as host.name and os as fallback
+exporters:
+  otlp:
+    endpoint: "ingest.{region}.signoz.cloud:443" # replace {region} with your region
+    tls:
+      insecure: false
+    headers:
+      "signoz-access-token": "<SIGNOZ_INGESTION_KEY>"
+  logging:
+    loglevel: debug
```
Comment (prashant-shahi) [inline docs/userguide/send-metrics-cloud.md:69]:
```suggestion
  logging:
    verbosity: detailed
```

File: docs/userguide/send-metrics-cloud.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
```
Comment (prashant-shahi) [inline docs/userguide/send-metrics-cloud.md:136]:
```suggestion
  logging:
    verbosity: detailed
```

same as above

File: docs/userguide/send-metrics-cloud.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+      process:
+       mute_process_name_error: true
+      processes: {}
+processors:
+  batch:
+    send_batch_size: 1000
+    timeout: 10s
+  # Ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md
+  resourcedetection:
+    detectors: [env, system, ec2] # include ec2 for AWS, gce for GCP and azure for Azure.
```
Comment (prashant-shahi) [inline docs/userguide/send-metrics-cloud.md:55]:
```suggestion
    detectors: [env, system, ec2] # include ec2 for AWS, gcp for GCP and azure for Azure.
```

**Review by prashant-shahi — CHANGES_REQUESTED** [review]
Left some comments requesting changes.

Also, I think there should be a section for collecting metrics using pod annotations in Kubernetes:

```yaml
podAnnotations:
    signoz.io/scrape: 'true'
    signoz.io/port: '8888'
    signoz.io/path: /metrics
```

**Comment by srikanthccv** [comment]
> Also, I think there should be a section for collecting metrics using pod annotations in Kubernetes:

Nothing in this document is about k8s. And k8s related thing deserves its own separate instructions. Asking for pod annotation seemed random on this change to me.

File: docs/userguide/send-metrics-cloud.md (hunk 3 above)
Comment (srikanthccv) [reply docs/userguide/send-metrics-cloud.md:115]: It's an example. Users can set whatever they want.

**Comment by prashant-shahi** [comment]
> Nothing in this document is about k8s. And k8s related thing deserves its own separate instructions. Asking for pod annotation seemed random on this change to me.

Okay, makes sense to not include it in this specific page.

But yeah, that should be included under metrics docs perhaps as a sub-page for easily collecting metrics in Kubernetes.

**Comment by srikanthccv** [comment]
@ankit01-oss does the onboarding workflow design consist of k8s env in metrics? I am not very well aware of what is covered in onboarding instructions. So if workflow by the environment such as k8s exists, then we should have the example docs as well.

**Pull request merged**